app/components/submission-list/index.js

```diff
@@ -5,28 +5,30 @@ import { observer } from '@ember/object';
 import ENUMS from 'irene/enums';
```

Comment:
Is there a reason why this component class isn't migrated?

Reply:
I migrated this but unit tests were not passing so reverted for now